### PR TITLE
fix: add Telegram icon and its relevant color

### PR DESCRIPTION
**Pull request by BSlaven**
Fix: #853 

I added a fix to the icon and text for telegram in this PR.
I only later realised that there is also an issue raised only for the icon problem.
I added the telegram icon in GetIcons.js file. 
Also added telegram light bluish color to config/links.json file.
Results:
Screenshot (43)

Hover: 
Screenshot (44)

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/857"></a>

### CODE REVIEW

**Review by Shivansh-yadav13 — APPROVED** [review]
LGTM 👍‍

**Review by emmalearnscode — APPROVED** [review]

**Comment by emmalearnscode** [comment]
@BSlaven merging a the tiktok issue created a conflict here. Could you merge the changes into your branch and push the update?

File: src/config/links.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "book": "#2196F3",
     "dollar": "#808080",
     "graduation-hat": "#808080",
+    "telegram": "#4995be",
     "tiktok": "#000000"
   }
-}
+}
```
Comment (Panquesito7) [inline src/config/links.json:30]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — APPROVED** [review]
Great work on this! 🚀 Thanks so much! 😄👍

**Pull request merged**